srcpkgs/minbif/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'minbif'
+pkgname=minbif
+version=1.0.5
+revision=1
+wrksrc="$pkgname-$version"
```

Comment:
not necessary, this is the default value. wkrsrc= should be set when the value differs from the default.